webview/src/setup/components/remotes/AmazonS3.tsx

```diff
@@ -24,7 +24,7 @@ export const AmazonS3 = () => (
       <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html">
```

Comment:
Out of scope of this pr, but noticed a validation error on nested `p` tags:

<img width="496" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/cdd0175c-4261-4592-b5e9-285f2ee48386">